extensions/tekton/generator-triggers/cmd/generate/generate.go

```diff
@@ -29,6 +29,8 @@ func main() {
 	// the CRD List types for which the model should be generated
 	// no other types need to be defined as they are auto discovered
 	crdLists := map[reflect.Type]schemagen.CrdScope{
+		reflect.TypeOf(triggers.Trigger{}):                   schemagen.Namespaced,
+		reflect.TypeOf(triggers.TriggerList{}):               schemagen.Namespaced,
```

Comment:
nit: it should be sufficient to include the *List

Reply:
I'm happy to change it!

Reply:
Aaaand done!